docs/examples/AutomaticMockGeneration.md

```diff
@@ -54,6 +54,11 @@ public class AutomaticMockGenerationExample
     public async Task SampleAsyncMethod()
     {
         await _dummyService.AsyncMethod();
+        await SamplePrivateMethod();
```

Comment:
This *should* cause the build to fail. You want to edit the `Unitverse.ExampleGenerator` project, which will update the content here - then this should be committed too. Have a look at the 'Build Protection' section in  https://mattwhitfield.github.io/Unitverse/contributing.html

Reply:
It didn't cause the build to fail... I will need to look into why that is 🤔